app/assets/javascripts/es6.js

```diff
@@ -0,0 +1,23 @@
+class Animal {
+  constructor(name) {
+    this.name = name;
+  }
+
+  speak() {
+    console.log(this.name + ' makes a noise.');
+  }
+}
+
+class Dog extends Animal {
+  speak() {
+    console.log(this.name + ' barks.');
+  }
+}
+
+class Cat extends Animal {
+  speak() {
+    console.log(this.name + " loves to chase cards and bite their tires like a dog, but isn't one.");
```

Comment:
Line 19 exceeds the maximum line length of 80.<br>Expected indentation of 6 space characters but found 4.<br>Unexpected console statement.